CSC8503CoreClasses/GuardObject.cpp

```diff
@@ -89,6 +89,8 @@ BehaviourAction* GuardObject::Patrol() {
 		}
 		else if (state == Ongoing) {
 			if (mCanSeePlayer == false) {
+				//std::cout << "Lost em";
```

Comment:
Maybe we can add a todo or another comment line to describe what why this line is commented out rather then deleting it ✌️